test/e2e/Tests/start-emulators.ps1

```diff
@@ -7,9 +7,6 @@ param(
     [Switch]
     $SkipStorageEmulator,
     [Parameter(Mandatory=$false)]
-    [Switch]
-    $StartCosmosDBEmulator,
```

Comment:
Were we ever using CosmosDBEmulator or are you just deleting unnecessary code?

Reply:
Just removing unnecessary code

Reply:
Cleaned up a few extra unused params as well